content/en/tracing/livetail.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Live Tail
+kind: documentation
+description: "See all your trace spans in real time."
+---
+
+# Live Tail
+
+{{< img src="tracing/live_tail/livetail_view.gif" alt="Live tail" >}}
+
+## Overview
+The [Live Tail][1] feature gives users the ability to see all trace spans in near real-time from anywhere in the Application. It displays spans as soon as they get out of the Datadog agent section and before they are indexed by Datadog, hence:
+All trace spans ingested by Datadog are displayed. (It’s a part of tracing without Limits*)
+Displayed spans have been processed.
+
+1. The span stream can be paused
+2. Write search parameters to refine the streaming view
+3. View the distributed trace in real-time
+4. Add or remove columns from span tags for a customized view 
+
+This feature allows you, for instance, to check if a process has correctly started, or if a new deployment went smoothly. Or view outage related information in real-time
+
+## Live Tail view
+
+Choose the `Live Tail` option in the time range selector to switch to the Live Tail view.
+The number of received spans per second is displayed at the top left, as well as the sampling rate. Since a stream of thousands of spans per second is not human readable, high throughput span streams are sampled for visual clarity.
```

Comment:
```suggestion
Find the number of received spans per second and the sampling rate at the top left. Since a stream of thousands of spans per second is not human readable, high throughput span streams are sampled for visual clarity.
```